public/css/common.less

```diff
@@ -328,3 +332,93 @@ div.show-more {
 .servicegropup-list > col.object-statistics-graph {
   padding-right: .25em;
 }
+
+.comment-popup {
+  font-size: 1.2em;
+  height: 6em;
+  border-radius: 0.25em;
+  border: @gray-light 1px solid;
+  background-color: @body-bg-color;
+
+  .comment-list .main {
+    border: none;
+    width: auto;
+    overflow: auto;
+
+    // This is necessary to limit the visible comment lines
+    // and to fit the image into the comment pop-up in the host/service detail view.
+    .caption {
+      height: 3em;
+
+      img {
+        max-height: 1.7em;
+      }
```

Comment:
* Why 1.7em? (Because it fits best isn't an explanation :wink: )
* Also, if you expect images not extending over one line of text in the popup, why not in the standard lists?

Reply:
Should I make them as small as the text `(12px max-height)`? Example below: 
![Screenshot 2021-08-19 at 11 04 52](https://user-images.githubusercontent.com/54990055/130041301-12e238ca-646e-48c9-a965-8222ed5dfb7a.png)

And same for `CommentController`. And then just show the original size in the comment detail, as it is at the moment.

Reply:
:+1: